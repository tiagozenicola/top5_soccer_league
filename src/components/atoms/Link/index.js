import Link from './style';


export default Link;
