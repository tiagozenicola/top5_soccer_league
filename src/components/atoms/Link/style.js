import styled from 'styled-components';


const Link = styled.button`
  color: blue;
  border: none;
  background: none;
  text-decoration: underline;
  cursor: pointer;
`;


export default Link;
