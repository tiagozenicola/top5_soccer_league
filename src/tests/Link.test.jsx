import React from 'react'
import renderer from 'react-test-renderer'
import enzyme, { mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Link from '../components/atoms/Link'
import 'jest-styled-components'

enzyme.configure({ adapter: new Adapter() })

it('renders Link correctly', () => {
  const link = renderer.create(<Link />).toJSON()
  expect(link).toMatchSnapshot()
})
