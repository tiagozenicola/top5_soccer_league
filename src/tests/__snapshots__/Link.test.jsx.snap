// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Link correctly 1`] = `
.c0 {
  color: blue;
  border: none;
  background: none;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

<button
  className="c0"
/>
`;
